integration-tests/package.json

```diff
@@ -13,10 +13,10 @@
     "author": "",
     "license": "Apache-2.0",
     "dependencies": {
-        "@frequency-chain/api-augment": "v0.9.29",
-        "@polkadot/api": "9.10.2",
-        "@polkadot/types": "9.10.2",
-        "@polkadot/util": "10.2.1",
+        "@frequency-chain/api-augment": "^v0.9.29",
+        "@polkadot/api": "^9.6.2",
+        "@polkadot/types": "^9.6.2",
+        "@polkadot/util": "^8.7.1",
```

Comment:
This looks like a reversion.

Reply:
I did this to pin the version on which integration tests run fine

Reply:
I see yes something updated since this PR let me keep whats in main